tests/scenarios/stage1-test.ts

```diff
@@ -32,7 +32,11 @@ appScenarios
       hooks.before(async assert => {
         process.env.THROW_UNLESS_PARALLELIZABLE = '1'; // see https://github.com/embroider-build/embroider/pull/924
         app = await scenario.prepare();
-        let result = await app.execute('cross-env STAGE1_ONLY=true node ./node_modules/ember-cli/bin/ember b');
+        let result = await app.execute('node ./node_modules/ember-cli/bin/ember b', {
+          env: {
+            STAGE1_ONLY: 'true',
```

Comment:
This was just opportunistic since I was in there.

There's no reason for our tests to need cross-env.